scheduler/src/cook/test/testutil.clj

```diff
@@ -565,4 +579,30 @@
                                     synthetic-pods-config ; synthetic-pods-config
                                     node-blocklist-labels ; node-blocklist-labels
                                     (Executors/newSingleThreadExecutor) ; launch-task-executor-service
-                                    )))
\ No newline at end of file
+                                    )))
+
+
+(defn make-and-write-kubernetes-compute-cluster
```

Comment:
Where is this used and can it be shared with the other similar implementations of it?

Reply:
The function is being used to make kubernetes compute clusters in the scheduler unit tests. Thanks for the suggestion, it's pretty similar to the function `make-kubernetes-compute-cluster` above it. I slightly modified the `make-kubernetes-compute-cluster` function and had `make-and-write-kubernetes-compute-cluster` call that function.